src/parser/rogue/assassination/modules/spells/shadowlands/legendaries/DuskwalkersPatch.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Events, { CastEvent } from 'parser/core/Events';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import { ASS_VEN_CDR_PER_ENERGY } from 'parser/rogue/shared/constants';
+
+class DuskwalkersPatch extends Analyzer {
+  static dependencies = {
+    spellUsable: SpellUsable,
+  };
+
+  cdrPerEnergy = ASS_VEN_CDR_PER_ENERGY;
```

Comment:
Is there a reason you are assigning the constant to a variable instead of just referring to the constant within the module?

like further down below you could just cooldownReductionMs = ASS_VEN_CDR_PER_ENERGY * this.lastEnergyCost

Just seems odd to have another variable be the middle man. If it is an issue of having a variable name that makes sense, you could just declare the constant within this module instead of in the constants file (If the constant is only used in this module and not any others).

Reply:
No. No reason except not noticing what I did there. This is gross. I'll fix it.